src/app/organizations/organization-details/organization-details.component.ts

```diff
@@ -109,13 +93,19 @@ export class OrganizationDetailsComponent implements OnInit {
                         text: 'Success!',
                         type: 'success'
                     }).then(function () {
-                        this.organizationService.getMembersForOrganization(orgID)
-                            .subscribe(res => {
-                                this.orgMembers = res.members;
-                                SpinnerUtil.hideSpinner();
-                            }, hideSpinnerAndDisplayError);
+                        self.refreshOrganizationMemberSubject.next();
+                        SpinnerUtil.hideSpinner();
                     }.bind(this));
```

Comment:
if you are already using `const self = this;`, why you still use `.bind(this)`? Either one is unnecessary. 

```
SpinnerUtil.showSpinner();
this.organizationService.removeMemberFromOrganization(orgID, memberID).pipe(
    this.handleErrors(),
    from(swal('Successfully deleted user from the organization', ...)),
    tap(() => self.refreshOrganizationMemberSubject.next()),
    finalize(() => SpinnerUtil.hideSpinner())
).subscribe();
```

This is the same functionality in a cleaner manner. Please, tell me if anything from here is not fully clear, because it's essential.